daemon/logger/logbuffer/ringbuff.go

```diff
@@ -18,26 +23,33 @@ type RingBuffer struct {
 	cap    int
 	closed bool
 	q      *queue
+
+	maxBytes     int64
+	currentBytes int64
 }
 
 // New creates new RingBuffer.
-func New(cap int) *RingBuffer {
+func New(cap int, maxBytes int64) *RingBuffer {
```

Comment:
I think we should remove the `cap` here since the buffer is controlled by the length of bytes, not `cap` here.